### ISSUE TITLE
[BUG] Source field is not shown in "review" (sentence validation)

### ISSUE THREAD

**Issue by HarikalarKutusu** [common-voice/common-voice#4580]
While testing the issue #4559, I found out that what we write in the source field in "write" is not shown on the "review" page. Two possibilities:
- After posting from "write", the data is not correctly written into the database
- The source data is already in the database, but it is not retrieved/displayed in "review"

This is kind of important, because according to the workflows set by MCV, if there is no valid source info, we should reject the sentence in "review". Which I did for a couple of sentences before investigating it :(

**Comment by jgrant29** [comment]
Thanks for posting this, @HarikalarKutusu.  We will look into adding source information on review pages to avoid submitting invalid sentences to the platform.

**Comment by amikeco** [comment]
It's September now, and the source is still not shown on the Review screen, while the column on the left tells us it's important to check the right citation to approve a sentence.

**Comment by HarikalarKutusu** [comment]
Next week v19.0 will be out and we will see if they are in the database or not :)

**Comment by moz-kathyreid** [comment]
@HarikalarKutusu Thank you as always for being so diligent - is this bug still occurring?

**Comment by HarikalarKutusu** [comment]
Yes it does. I have a fix but did not PR it. Please contact me if you want to know why :)

**Comment by HarikalarKutusu** [comment]
This turns out to be a simple bug caused by bad Fluent call. It seems to be left-over from transition to multiple translation files.

Existing form:
```
      <Localized id="sc-my-source" vars={{ source }}>
        <p className="source" />
      </Localized>
```

In the new order, this actually should be:
```
      <Localized id="sc-review-form-source" vars={{ sentenceSource: source }}>
        <p className="source" />
      </Localized>
```

This is defined in `review.ftl` as:
```
sc-review-form-source = Source: { $sentenceSource }
```

Old form was (from the left over German translation file):
```
sc-my-source = Quelle: { $source }
```
Sending a PR...

**Comment by moz-dfeller** [comment]
Good catch @HarikalarKutusu, I merged the your fix #4854 . Closing this issue now 👍
